### ISSUE TITLE
Implementing split view

### ISSUE THREAD

**Issue by johnmara** [mozilla/pdf.js#7131]
Hello,
I'm new to pdf.js but I found it interesting. I have to choose an open source project and make changes for my uni, so I chose this one. I want to begin with something easy so I am thinking to  implement a split view for a document. So you can see two pages in parallel or even documents.
I'll try to implement it with my friend.

**Comment by Snuffleupagus** [comment]
Closing as duplicate of issue #590. (As indicated there, and in the PRs that tried to implement it, this feature is *not* an easy project to take on.)

**Comment by johnmara** [comment]
Could you suggest me any other easy to implement feature? At first I thought of adding annotations but I found out that it already supports viewing annotations(but not creating them, even there is a nice video on youtube https://www.youtube.com/watch?v=agrBJH1TriI ). 
Thank you in advance :D

**Comment by timvandermeij** [comment]
Nice to hear that you are interested in helping us develop PDF.js! I don't know if you need a small or a big task, but we have a "good beginner bug" category containing issues that you can work on that do not require too much knowledge of the codebase. You can find the category at https://github.com/mozilla/pdf.js/issues?q=is%3Aopen+is%3Aissue+label%3A5-good-beginner-bug.

Feel free to contact us on IRC if you need more information.

**Comment by iago-pssjd** [comment]
@Snuffleupagus @timvandermeij This issue does seem neither a duplicate of #590, nor solved by #9208. That issue and PR seems to refer to view 2 continuous pages in parallel, but it would be great pdf.js allowing to see two arbitrary pages in parallel, needed, for example, reading a book or a paper and their references, etc., or also reading two different pdf's side by side.
